packages/webviz-core/package.json

```diff
@@ -24,6 +24,7 @@
     "moment": "2.22.2",
     "moment-duration-format": "2.2.2",
     "moment-timezone": "0.5.23",
+    "monaco-editor": "0.17.1",
```

Comment:
Since we have this already in the global package.json, we should probably not also have it here, because then we might get multiple instances of this package, which can cause confusing issues.